lib/run-status.js

```diff
@@ -134,9 +135,10 @@ RunStatus.prototype.prefixTitle = function (file) {
 	var separator = ' ' + chalk.gray.dim(figures.pointerSmall) + ' ';
 
 	var prefix = path.relative('.', file)
-		.replace(this.base, '')
+		.replace(new RegExp('^' + escapeStringRegexp(this.base)), '')
 		.replace(/\.spec/, '')
 		.replace(/\.test/, '')
+		.replace(/__tests__\//, '')
```

Comment:
Might be better to add a filter after splitting:

```js
.split(path.sep)
.filter(function (component) {
  return component !== '__tests__';
})
.join(separator);
```

That way you don't have to worry about the slashes.